### DISCUSSION TITLE
Block Code with revealjs

### DISCUSSION THREAD

**Discussion by friosavila** [quarto-dev/quarto-cli#4943] [Q&A] [answered]
Hi, 
Im wondering if its possible (and how) to select the height allowed for a code block

Specifically, say that I'm presenting a rather long code (for any language)

```
1
1+1
1+2
2+3
... etc
```
Right now, I see that when using HTML documents, the code could be as long as necessary. 
But when using revealjs, block codes are limited in height to be about 18 lines (when language is defined)
but only 14 lines when there is no language specified. 
Is there a way to increase the number of lines shown with revealjs?

Thank you

**Comment by mcanouil** [comment, marked as answer]
You can play with height/max-height depending on what you want.

```css
.reveal pre.sourceCode code {
  /* height: 500px; */
  max-height: 1000px;
}
```
<img width="1624" alt="image" src="https://user-images.githubusercontent.com/8896044/227741775-8cb0c2ba-bb92-4119-9aca-8b570ba962bf.png">
<img width="1624" alt="image" src="https://user-images.githubusercontent.com/8896044/227741789-c3edea64-f6bd-46ec-bde4-ac7cae2d3bdd.png">

You can use your favourite browser inspect/developper mode to see what CSS classes/attributes to tweak.
<img width="1508" alt="image" src="https://user-images.githubusercontent.com/8896044/227741875-bc0cc778-eece-49d0-a5cd-261a2c1727cf.png">

**Comment by friosavila** [comment]
Thank you .
This is perfect!

**Comment by mcanouil** [comment]
FYI:
- #2073
